lib/dashboard/dashboard_card.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter_tindercard/flutter_tindercard.dart';
+import 'package:flutter/material.dart';
+
+List<String> dummyImages = const [
+  "images/DashboardBg.png",
+  "images/DefaultBg.png",
+  "images/DashboardBg.png",
+  "images/DefaultBg.png", 
+  "images/DashboardBg.png"
+];
+
+class DashboardCard extends StatefulWidget {
+  @override
+  _DashboardCardState createState() => _DashboardCardState();
+}
+
+class _DashboardCardState extends State<DashboardCard> 
```

Comment:
don't think this should be prefixed with an underscore

https://dart.dev/guides/language/effective-dart/style#dont-use-a-leading-underscore-for-identifiers-that-arent-private

Reply:
yup you right my b